core/src/main/java/com/netflix/conductor/service/ExecutionService.java

```diff
@@ -133,11 +164,23 @@ public Task poll(String taskType, String workerId, String domain) throws Excepti
 		return tasks;
 	}
 
-	public List<PollData> getPollData(String taskType) throws Exception{
+	public Task getLastPollTask(String taskType, String workerId, String domain) throws Exception{
+		List<Task> tasks = poll(taskType, workerId, domain, POLL_COUNT, POLLING_TIMEOUT_IN_MS);
```

Comment:
POLL_COUNT is not a good constant name, either use 1 directly here or name the constant POLL_COUNT_ONE